### PR TITLE
update profile

### PR DESCRIPTION
**Pull request by mercur0us**

### CODE REVIEW

File: personal_page/kayleen_howen.md [hunk 1]
```diff
@@ -9,28 +9,35 @@ show_sidebar: false
...
-
+<img src="{{site.url}}/img/kayleen_howen.jpg" align="left" hspace="10" width="25%">
+</br>
+An Indonesian junior student currently studying agronomy at National Taiwan University. Eager to learn but procastinates too much somtimes. 
+<br>
+
+#### Contact:
+><i class="fas fa-at"></i> [Email](mailto:kayleenhowen@gmail.com)
+ <br clear="all">
+><i class="fab fa-github"></i> [Github](mercur0us)
```
Comment (stevenhwu) [inline personal_page/kayleen_howen.md:20]:
```suggestion
><i class="fab fa-github"></i> [Github](https://github.com/mercur0us)  
```

**Review by stevenhwu — CHANGES_REQUESTED** [review]
I think that might be better to make GitHub a proper link.

File: personal_page/kayleen_howen.md (hunk 1 above)
Comment (stevenhwu) [reply personal_page/kayleen_howen.md:20]: @mercur0us Ok with this?

**Comment by mercur0us** [comment]
sorry for the late reply. Yes that's ok!

**Review by stevenhwu — APPROVED** [review]

**Pull request merged**